screens/Menus/MenuScreen.js

```diff
@@ -312,6 +311,9 @@ function Menu(props) {
             </View>
           </View>
         ))}
+        <Text style={styles.title}>
+          {'Lista de compras'}
+        </Text>
```

Comment:
Acá podría ser directamente el texto: 

```suggestion
        <Text style={styles.title}>
          Lista de compras
        </Text>
```